code/frameworks/angular/src/builders/start-storybook/index.ts

```diff
@@ -132,10 +132,10 @@ async function setup(options: StorybookBuilderOptions, context: BuilderContext)
   };
 }
 function runInstance(options: StandaloneOptions) {
-  return new Observable<void>((observer) => {
+  return new Observable<number>((observer) => {
     // This Observable intentionally never complete, leaving the process running ;)
     buildDevStandalone(options as any).then(
-      () => observer.next(),
+      (info: { port: number }) => observer.next(info?.port),
```

Comment:
Is this correct? Shouldn't it rather be:

```suggestion
      ({ port }) => observer.next({ port }),
```